src/content/docs/style-guide/article-templates/create-release-notes.mdx

```diff
@@ -23,3 +23,4 @@ Review the recommended [best practices for New Relic release notes](https://nerd
 
 A Tech Docs hero will review your release note content and approve your pull request to get it published.
 
+If release notes are blockers for your team, please be aware that release note pull requests may take a few hours to appear on the docs site after merging. 
```

Comment:
Can we re-frame this to be more active/positive rather than "please be aware"? e.g.

> We only build and deploy the site a few times a day, and currently builds can take a few hours to complete. If your release is time-sensitive, ensure you've budgeted enough time to get the docs live.

Reply:
done